pkg/gofr/http/request_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"context"
 	"errors"
+	"github.com/gorilla/mux"
```

Comment:
Fix import order please

Import out of std lib are separated from std lib ones with a line feed